server/routes.js

```diff
@@ -1,14 +1,29 @@
 // var Database = require ('./database/config.js');
 
 var router = require('express').Router();
+var bcrypt = require('bcrypt');
+var SALT_WORK_FACTOR = 10;
 
 // Add new user account to database
 // app.post('/api/addnewuser', function(req, res){
+//   var hash = bcrypt.hashSync(req.body.password, SALT_WORK_FACTOR);
```

Comment:
Has any of this code been tested and confirmed to work? If so, why is it currently commented out? I'm don't think we should have commented out code in our Master repo/branch.